web/src/recent_view_ui.ts

```diff
@@ -1318,6 +1326,12 @@ export function show(): void {
         set_visible: recent_view_util.set_visible,
         complete_rerender,
     });
+    last_scroll_offset = undefined;
+
+    if (reattach_event_handlers) {
+        assert(topics_widget !== undefined);
+        topics_widget.set_up_event_handlers();
+    }
 
     if (onboarding_steps.ONE_TIME_NOTICES_TO_DISPLAY.has("intro_recent_view_modal")) {
         const html_body = render_introduce_zulip_view_modal({
```

Comment:
Does this PR close anything from https://github.com/zulip/zulip/issues?q=is%3Aissue+is%3Aopen+%22recent+topics%22, such as #17933?

Reply:
Added to commit message:

Fixes #17933, #27517